gitlab_migration.md

```diff
@@ -91,6 +91,7 @@ You also will likely want to just get the `build` and/or `test` stages passing f
 1. Copy the [Java Tomcat Manager GitLab template][gitlab-java-tomcat] into your project, renaming it
    to `.gitlab-ci.yml`
 1. Update the template to match the needs of the project.
+1. Add sensitive [environment variables][gitlab-env-vars] such as `TOMCAT_MANAGER_STAGING_PASSWORD` and others as needed to the GitLab project.
 
```

Comment:
Where those sensitive environment variables like `TOMCAT_MANAGER_STAGING_PASSWORD`are defined?

Reply:
@lsitu - it was just an example referencing the java example `.gitlab-ci.yml` file. So they could be anything you'd need to define that's sensitive. They would be setup similar to the SSH private key noted earlier on the page, but I wanted to provide something more generic with a link to the GitLab docs.

How could I make this worded better or more useful as a reference? Happy to make any changes as needed :)

Reply:
@mcritchlow I think defining all variables in .gitlab-ci.yml are fine. Just want to know whether those environment variables are `sensitive` that requires us to set them up differently or not since you mention the word `sensitive` above. Thanks.

Reply:
@lsitu - Ah, I see. Yes it's really important that the sensitive environment variables are defined, and their values stored, in the project settings so they aren't leaked during CI/CD runs. Do you think the docs could use better clarification around that? It's definitely important we get that right.

Reply:
@mcritchlow Maybe we need to clarify that since we need those setting migrated to Giblab CI as well. But if we don't need to use any such `sensitive` environment variable for Giblab CI at all, I think we can skip it. What do you think?

Reply:
@lsitu - I think we'll definitely need sensitive environment variables for some repos. Capistrano apps will need the SSH private key set so it's available in CI. The java apps that can be deployed via Tomcat Manager will need the account credentials/pw set as sensitive env vars as well. So it sounds like the way I worded things is perhaps not clarifying this properly. I'll take another pass at it. If you have any suggestions, please let me know :)